packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -776,73 +775,14 @@ router.post(
   })
 );
 
-/** ---------------------------------------------------------------------------
- * POST /internal/patient/ops/coverage-assessment/validate
- *
- */
-router.post(
-  "/ops/coverage-assessment/validate",
-  requestLogger,
-  asyncHandler(async (req: Request, res: Response) => {
-    const payload = coverageAssessmentValidationSchema.parse(req.body);
-
-    const invalidPatients: { patient: unknown; errors: string[] }[] = [];
-    payload.patients.map(patient => {
-      const errors: string[] = [];
-      // DOB
-      if (!patient.dob) {
-        errors.push("Missing DOB");
-      } else {
-        const dob = normalizeDate(patient.dob);
-        if (!dob) errors.push("Invalid DOB");
-      }
-      // Gender
-      if (!patient.gender) {
-        errors.push("Missing Gender");
-      } else {
-        const gender = normalizeGender(patient.gender);
-        if (!gender) errors.push("Invalid Gender");
-      }
-      // Name
-      if (!patient.firstname) errors.push("Missing First Name");
-      if (!patient.lastname) errors.push("Missing Last Name");
-      // Email
-      if (patient.email1 && !isEmailValid(patient.email1)) errors.push("Invalid Email 1");
-      if (patient.email2 && !isEmailValid(patient.email2)) errors.push("Invalid Email 2");
-      // Phone
-      if (patient.phone1 && !isPhoneValid(patient.phone1)) errors.push("Invalid Phone 1");
-      if (patient.phone2 && !isPhoneValid(patient.phone2)) errors.push("Invalid Phone 2");
-      // Zip
-      if (!patient.zip) {
-        errors.push("Missing Zip");
-      } else {
-        const zip = normalizeZipCodePassive(patient.zip);
-        if (!zip) errors.push("Invalid Zip");
-      }
-      // City
-      if (!patient.city) errors.push("Missing City");
-      // State
-      if (!patient.state) {
-        errors.push("Missing State");
-      } else {
-        const state = normalizeState(patient.state);
-        if (!state) errors.push("Invalid State");
-      }
-      // Address
-      if (!patient.addressline1) errors.push("Missing Address Line");
-      if (errors.length > 0) invalidPatients.push({ patient, errors });
-    });
-
-    return res.status(status.OK).json({ invalidPatients });
-  })
-);
-
 /** ---------------------------------------------------------------------------
  * POST /internal/patient/ops/coverage-assessment
  *
  * return the coverage
  * @param req.query.cxId The customer ID.
  * @param req.params.id The patient ID.
+ * @param req.query.facilityId The facility ID for running the coverage assessment.
+ * @param req.query.dryrun Whether to simply validate or run the assessment.
```

Comment:
It'd be good to explicitly indicate this is optional and what the default behavior is.

Reply:
Done.